Meter.c

```diff
@@ -49,16 +50,22 @@ static inline void Meter_displayBuffer(const Meter* this, RichString* out) {
 /* ---------- TextMeterMode ---------- */
 
 static void TextMeterMode_draw(Meter* this, int x, int y, int w) {
+   assert(x >= 0);
+   assert(w <= INT_MAX - MAXIMUM(0, x));
```

Comment:
```suggestion
   assert(w <= INT_MAX - x);
```

Suffices. `x>=0` has just been checked above.